### ISSUE TITLE
Should incomingUni/BidirectionalStreams be "owning"?

### ISSUE THREAD

**Issue by jan-ivar** [w3c/webtransport#507]
From https://github.com/w3c/webtransport/issues/424#issuecomment-1270923443 and https://github.com/w3c/webtransport/issues/424#issuecomment-1271454749:
> > Attempting to transfer `wt.incomingUndirectionalStreams` doesn't appear to work, as you say.  When `getReader()` is called on the transferred readable stream of streams, the error is `A ReadableStream could not be cloned because it was not transferred.` 
> 
> Sorry about this. Currently when a stream is transferred, the chunks that are passed through the stream are always cloned, never transferred. Since `incomingUnidirectionalStreams` is a stream of streams, each chunk is a stream, and streams can only be transferred, not cloned.
> 
> There was a plan to also support transferring the chunks (see [Future Work](https://github.com/whatwg/streams/blob/main/transferable-streams-explainer.md#future-work)) but without a concrete use case it never proceeded to the design stage.

https://github.com/whatwg/streams/pull/1271 adds a new `"owning"` ReadableStream type, which allows for transferable chunks, albeit for a different purpose.

@ricea would making [incomingUnidirectionalStreams](https://w3c.github.io/webtransport/#dom-webtransport-incomingunidirectionalstreams) and [incomingBidirectionalStreams](https://w3c.github.io/webtransport/#dom-webtransport-incomingunidirectionalstreams) `"owning"` solve the `DataCloneError`, or is that a different part of the spec?

**Comment by ricea** [comment]
Currently the `"owning"` logic isn't wired up to the transferable streams logic. Once that is done, then it should work.

**Comment by jan-ivar** [comment]
@ricea Do we have an issue open for that? Would it be different from https://github.com/whatwg/streams/issues/1124?
